app/controllers/bookmarked_posts_controller.rb

```diff
@@ -0,0 +1,31 @@
+class BookmarkedPostsController < ApplicationController
+  def show
+    @bookmark_ids = BookmarkedPost.where(user: current_user)
+    @bookmark_posts = []
+    @bookmark_ids.each do |x|
+      @bookmark_posts << Post.find(x.post_id)
+    end
+  end
+
+  def update
+    @post = Post.find(params[:id])
+    @user = current_user
+    status = 200
+    data = nil
+    @bookmark = BookmarkedPost.find_by(user: @user, post: @post)
+    if @bookmark != nil
```

Comment:
.blank? - palasi par to

Reply:
Oh, es tur no sakuma centos ielikt, visvisadi meginaju nesagaja, tapec atstaju veco, bet tagad pameginaju vel caur rails console paskatities kas tur ka , un beigas sanaca viss :)